regimes/gr/extensions.go

```diff
@@ -0,0 +1,306 @@
+package gr
+
+import (
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+)
+
+// Regime extension codes.
+const (
+	ExtKeyIAPRVATCat    = "gr-iapr-vat-cat"
+	ExtKeyIAPRExemption = "gr-iapr-exemption"
+)
+
+var extensionKeys = []*cbc.KeyDefinition{
+	{
+		Key: ExtKeyIAPRVATCat,
+		Name: i18n.String{
+			i18n.EN: "VAT category",
+			i18n.EL: "Κατηγορία ΦΠΑ",
+		},
+		Codes: []*cbc.CodeDefinition{
+			{
+				Code: "1",
+				Name: i18n.String{
+					i18n.EN: "VAT rate 24%",
```

Comment:
Should numbers be listed in the name? I'd go with the standard names I think and avoid numbers to reduce issues when they change.

Reply:
I used the same descriptions that appear in the myDATA docs (both in Greek and English versions) for consistency with the specs. But it's true those will become outdated. I'll rename them to use the rate names.

Reply:
Addressed in [512ab72](https://github.com/invopop/gobl/pull/307/commits/512ab72a85133320e6986ac3f1e59dc2fab0285c)